src/tunacode/cli/commands/implementations/conversation.py

```
@@ -0,0 +1,115 @@
+                elif isinstance(msg, dict):
+                    if msg.get("role") == "assistant" and msg.get("content"):
+                        summary_text = msg["content"]
+                        break
+                # Handle other message types
+                elif hasattr(msg, "content") and hasattr(msg, "role"):
+                    if getattr(msg, "role", None) == "assistant":
+                        summary_text = msg.content
+                        break
+
+                if summary_text:
+                    break
```

Pylint (3.3.7)

[refactor] Too many nested blocks (8/5)

(R1702)